src/client/app/containers/editor.mode.ts

```diff
@@ -125,6 +130,12 @@ export class EditorMode {
 
     showImport$ = this._store.select(fromRoot.getImportState);
 
+    ngOnDestroy() {
+        this.snippetSub.unsubscribe();
+        this.sharingSub.unsubscribe();
+        this.errorsSub.unsubscribe();
```

Comment:
Can you check for null on all of these before unsubscribing?

Reply:
oops, fixed.